YvesIraguha/personalFinanceMobile
# #171621694 Separate Expenses From Home Page

#### What does this PR do?
- Allow a user to view menu
#### Description of Task to be completed?
-  Build the home drawer 
#### How should this be manually tested?
- Clone the repo and checkout the branch `ft-separate-expenses-from-home-page-#171621694 `
- checkout the respective branch, and run `yarn && yarn start` to start the application command
- Follow the directions provided to open the application in the simulator or real device
-  Ensure that your back-end is working. 
- Click on the profile avatar to access the home navigation drawer
#### What are the relevant pivotal tracker stories?
#171621694
